server/src/testFixtures/java/io/crate/testing/QueryTester.java

```diff
@@ -86,9 +86,24 @@ public Builder(Path tempDir,
                        Version indexVersion,
                        String createTableStmt,
                        AbstractModule... additionalModules) throws IOException {
-            // Disable OID generation for columns/references in order to be able to compare the query outcome with
-            // expected ones.
-            LongSupplier columnOidSupplier = () -> COLUMN_OID_UNASSIGNED;
+            this(tempDir,
+                threadPool,
+                clusterService,
+                indexVersion,
+                createTableStmt,
+                // Disable OID generation for columns/references in order to be able to compare the query outcome with
+                // expected ones.
+                ()-> COLUMN_OID_UNASSIGNED,
```

Comment:
Not sure, but checkstyle may complain.
```suggestion
                () -> COLUMN_OID_UNASSIGNED,
```